src/MongoDB/BulkWrite.c

```diff
@@ -315,7 +315,7 @@ HashTable *php_phongo_bulkwrite_get_debug_info(zval *object, int *is_temp TSRMLS
 		add_assoc_null_ex(&retval, ZEND_STRS("collection"));
 	}
 
-	add_assoc_bool_ex(&retval, ZEND_STRS("ordered"), intern->bulk->ordered);
+	add_assoc_bool_ex(&retval, ZEND_STRS("ordered"), intern->bulk->flags.ordered);
```

Comment:
flags also contains bypassDocumentValidation.
I imagine you'd want to add that here too as part of https://jira.mongodb.org/browse/PHPC-443